lib/plugins/create/templates/aws-fsharp/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#build handlers
+dotnet restore
+dotnet publish -c release
+
+#install zip
+apt-get -qq update
+apt-get -qq -y install zip
+
+#create deployment package
+pushd bin/Release/netcoreapp1.0/publish
```

Comment:
@slang25 I wonder, rather than lines 7-14, could we instead use:

```
dotnet lambda package --configuration Release --framework netcoreapp1.0
```

As far as I can see the zip file it creates is just the /publish folder anyway